nautobot/core/checks.py

```diff
@@ -41,6 +44,17 @@ class E006(Error):
     obj = settings
 
 
+E007 = Error(
+    "CRYPTOGRAPHY_KEY must be set to a non-null value that differs from SECRET_KEY."
```

Comment:
Here you're stating that the crypto key should be different from the secret key, but it doesn't appear that we are enforcing that. If we don't wish to enforce it, that's fine (especially given a null crypto key inherits a derived key from secret key), I just wanted to make sure it wasn't overlooked.

Reply:
Because of the way `django-cryptography` is implemented, `CRYPTOGRAPHY_KEY = None` is equivalent to `CRYPTOGRAPHY_KEY = SECRET_KEY`. Either of those two cases should be caught by this check.